spring-grpc-spring-boot-autoconfigure/src/main/java/org/springframework/grpc/autoconfigure/server/GrpcServerReflectionAutoConfiguration.java

```diff
@@ -0,0 +1,30 @@
+package org.springframework.grpc.autoconfigure.server;
+
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import io.grpc.BindableService;
+import io.grpc.protobuf.services.ProtoReflectionService;
+
+/**
+ * {@link EnableAutoConfiguration Auto-configuration} for gRPC Reflection service
+ * <p>
+ * grpc.reflection.enabled=true must be present in configuration in order for the
+ * auto-configuration to execute
+ *
+ * @author Haris Zujo
+ */
+@Configuration
+@ConditionalOnClass(BindableService.class)
```

Comment:
We will want to guard this a little bit wider as the `ProtoReflectionService` is optional. We can replace w/:
```suggestion
@ConditionalOnClass(ProtoReflectionService.class)
```